test/controllers/issueish-list-controller.test.js

```diff
@@ -85,22 +85,40 @@ describe('IssueishListController', function() {
     assert.deepEqual(view.prop('issueishes').map(issueish => issueish.getNumber()), [11, 13, 12]);
   });
 
-  it('opens reviews', async function() {
-    const atomEnv = global.buildAtomEnvironment();
-    sinon.stub(atomEnv.workspace, 'open').resolves();
-    sinon.stub(reporterProxy, 'addEvent');
-    const pr = createPullRequestResult({number: 1337});
-    Object.assign(pr.repository, {owner: {login: 'owner'}, name: 'repo'});
-    const wrapper = shallow(buildApp({
-      workspace: atomEnv.workspace,
-      endpoint: getEndpoint('github.com'),
-      results: [pr],
-    }));
+  describe('openOnGitHub', function() {
+    const url = 'https://github.com/atom/github/pull/2084';
+
+    it('calls shell.openExternal with specified url', async function() {
+      const wrapper = shallow(buildApp());
+      sinon.stub(shell, 'openExternal').callsArg(2);
+
+      await wrapper.instance().openOnGitHub(url);
+      assert.equal(shell.openExternal.args[0][0], url);
```

Comment:
Writing it like below would be a bit more readable:
```suggestion
      assert.isTrue(shell.openExternal.calledWith(url));
```